Duplicati/Library/Backend/Jottacloud/Jottacloud.cs

```diff
@@ -222,6 +254,64 @@ public IEnumerable<IFileEntry> List()
             }
         }
 
+        /// <summary>
+        /// Retrieves info for a single file (used to determine file size for chunking)
+        /// </summary>
+        /// <param name="remotename"></param>
+        /// <returns></returns>
+        public IFileEntry Info(string remotename)
+        {
+            var doc = new System.Xml.XmlDocument();
+            try
+            {
+                // Send request and load XML response.
+                var req = CreateRequest(System.Net.WebRequestMethods.Http.Get, remotename, "", false);
+                var areq = new Utility.AsyncHttpRequest(req);
+                using (var resp = (System.Net.HttpWebResponse)areq.GetResponse())
```

Comment:
Is this statement needed?  The return value isn't used, and the following line appears to read the response.

Reply:
Nice catch! It seems to work fine without the extra GetResponse() call.